tests/foreman/cli/test_repository.py

```diff
@@ -691,3 +699,162 @@ def test_positive_upload_content(self):
+        result = ssh.command(
+            'semanage fcontext -a -t foreman_var_run_t "/mnt/{0}(/.*)?"'
+            .format(self.export_dir)
+        )
+        self.assertEqual(result.return_code, 0)
+
+        result = ssh.command('restorecon -Rv /mnt/{0}'.format(self.export_dir))
+        self.assertEqual(result.return_code, 0)
+
+        # Assert that we have the correct policy
+        result = ssh.command('ls -Z /mnt/ | grep {0}'.format(self.export_dir))
+        self.assertEqual(result.return_code, 0)
+        self.assertGreaterEqual(len(result.stdout), 1)
+        self.assertIn(
+            'unconfined_u:object_r:foreman_var_run_t:s0', result.stdout[0])
+
+        # Update the 'pulp_export_destination' settings to new directory
+        Settings.set({
+            'name': 'pulp_export_destination',
+            'value': '/mnt/{0}'.format(self.export_dir),
+        })
+
+    def tearDown(self):
+        """Remove the export directory with all exported repository files"""
+        result = ssh.command('rm -rf /mnt/{0}'.format(self.export_dir))
+        self.assertEqual(result.return_code, 0)
+
+    @tier3
+    def test_positive_export_custom_product(self):
+        """Export a repository from the custom product
+
+        @Feature: Repository - Export
+
+        @Assert: Repository was successfully exported, rpm files are present on
+        satellite machine
+        """
+        # Create organization, product and repository
+        org = make_org()
+        product = make_product({'organization-id': org['id']})
+        repo = make_repository({
+            'download-policy': 'immediate',
+            'organization-id': org['id'],
+            'product-id': product['id'],
+        })
+        repo_export_dir = '/mnt/{0}/{1}-{2}-{3}/{1}/{4}/custom/{2}/{3}'.format(
+            self.export_dir,
+            org['label'],
+            product['label'],
+            repo['label'],
+            ENVIRONMENT,
+        )
+
+        # Export the repository
+        Repository.export({'id': repo['id']})
+
+        # Verify export directory is empty
+        result = ssh.command('ls -l {0} | grep .rpm'.format(repo_export_dir))
+        self.assertEqual(len(result.stdout), 0)
+
+        # Synchronize the repository
+        Repository.synchronize({'id': repo['id']})
+
+        # Export the repository once again
+        Repository.export({'id': repo['id']})
+
+        # Verify RPMs were successfully exported
+        result = ssh.command('ls -l {0} | grep .rpm'.format(repo_export_dir))
+        self.assertEqual(result.return_code, 0)
+        self.assertGreaterEqual(len(result.stdout), 1)
+
+    @tier3
+    def test_positive_export_rh_product(self):
+        """Export a repository from the Red Hat product
+
+        @Feature: Repository - Export
+
+        @Assert: Repository was successfully exported, rpm files are present on
+        satellite machine
+        """
+        # Create organization, enable RH repository
+        org = make_org()
```

Comment:
The organization creation can be moved to the setUp method. Since this is a repeating code for all tests within this test case class.